src/app/components/map/map.component.ts

```diff
@@ -144,29 +151,38 @@ export class MapComponent implements OnInit, OnDestroy {
     };
 
     const subscription = this.geoDataService.features.subscribe((collection) => {
+      this.featuresList = collection;
       this.features.clearLayers();
       this.overlays.clearLayers();
+      this.nonPointFeatures = [];
       const markers = L.markerClusterGroup({
         iconCreateFunction: (cluster) => {
           return L.divIcon({html: `<div><b>${cluster.getChildCount()}</b></div>`, className: 'marker-cluster'});
         }
       });
 
+
       collection.features.forEach(d => {
         let feat: LayerGroup;
-        if (d.geometry.type === 'Polygon' && d.properties.style) {
+
+        d.properties.style = d.properties.customStyle
```

Comment:
related to https://github.com/TACC-Cloud/hazmapper/pull/86/files#r852545813, I still don't think we should set the style on the feature collection as then its shown to the user but isn't reflective of the data.